string_lab.js

```diff
@@ -0,0 +1,148 @@
+// # String Exercises
```

Comment:
Good demonstration of how to use string methods and properties to solve problems. In the future assignments, I'd like to see you solving bonus questions as they are good extra practice to improve your skill. Keep up the good work!